addons/project/static/src/scss/project_dashboard.scss

```diff
@@ -60,6 +60,18 @@
             visibility: visible;
         }
     }
+
+    .o_project_kanban_bottom {
+        flex-wrap: wrap;
+
+        .o_project_kanban_bottom_row {
+            flex-basis: 100%;
+        }
+
+        .o_project_kanban_bottom_left {
+            flex-basis: 80%;
+        }
+    }
```

Comment:
See my comment where you use these classes. If you apply my suggestion we could normally remove the classes. 🙂